tokio/src/runtime/blocking/pool.rs

```diff
@@ -82,6 +83,26 @@ pub(crate) enum Mandatory {
     NonMandatory,
 }
 
+pub(crate) enum SpawnError {
+    /// Pool is shutting down and the task was not scheduled
+    ShuttingDown,
+    /// There are no worker threads available to take the task
+    /// and the OS failed to spawn a new one
+    NoThreads(io::Error),
+}
+
+#[allow(clippy::from_over_into)] // Orphan rules
```

Comment:
Hm. Orphan rule shouldn't apply to an `impl From<T>` where T is a local type?|

Reply:
Oh you're totally right, I think I've internalized an overly strict set of the orphan rules :sweat_smile: